docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "2"
+services:
+  nfldb:
+    build:
+      context: .
+    depends_on:
+      - "postgres"
+    restart: always
+    command: ['./install/wait-for-postgres.sh', 'postgres', './scripts/nfldb-update', '--interval', '60' ]
```

Comment:
@bhavik3jain my question was related to this line. I was asking if this is a good default command to have for `nfldb-update`